### ISSUE TITLE
lens distortion

### ISSUE THREAD

**Issue by mesnaash** [chulminy/CIVE497-CIVE700#65]
Can we say that the rotation and translation matrices in the projection matrix is due to the radial and tangential distortion of the camera lens, receptively?

**Comment by chulminy** [comment]
No!. Projection matrix only works when there is no lens distortion on your image.

**Comment by mesnaash** [comment]
Thanks
